contrib/ubase/patches/login.patch

```diff
@@ -0,0 +1,84 @@
+Index: ubase/ubase/login.c
+===================================================================
+--- ubase.orig/ubase/login.c	2023-05-30 17:51:24.466728782 +0200
++++ ubase/ubase/login.c	2023-05-30 17:54:52.876734143 +0200
+@@ -10,45 +10,21 @@
+ #include <string.h>
+ #include <time.h>
+ #include <unistd.h>
+-#include <utmp.h>
```

Comment:
In `login` we have this `writeutmp` function which is called after a successful login ([ubase/login.c](https://git.suckless.org/ubase/file/login.c.html#l120)). However in Mimiker we don't have `ttyname` implemented ([ttyname.c](https://github.com/cahirwpz/mimiker/blob/master/lib/libc/gen/ttyname.c)) which is necessary for `writeutmp` to work correctly, so I needed to remove the `writeutmp` call and its definition (otherwise we get ` error: unused function` during compilation).

Reply:
What about fixing output from `ttyname` as follows:
```c
if (!tty)
    tty = "unknown";
```

Reply:
I added `tty = "unknown";` and ported `writeutmp` in https://github.com/cahirwpz/mimiker/pull/1371/commits/369a272be9c5051134d5dbcc9445fb20185fddf4.